Source/BusinessLogic/Jobs/BoardGameGeekCleanUpService/BoardGameGeekCleanUpService.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.RegularExpressions;
+using BoardGameGeekApiClient.Interfaces;
+using BoardGameGeekApiClient.Models;
+using BusinessLogic.DataAccess;
+using BusinessLogic.Models;
+using BusinessLogic.Models.User;
+using RollbarSharp;
+
+namespace BusinessLogic.Jobs.BoardGameGeekCleanUpService
+{
+    public class BoardGameGeekCleanUpService : IBoardGameGeekCleanUpService
+    {
+        private const string CleanYearPattern = @"\w*\(\d{4}\)";
+
+        private readonly IDataContext _dataContext;
+        private readonly IBoardGameGeekApiClient _boardGameGeekApiClient;
+        private readonly IRollbarClient _rollbar;
+
+        public BoardGameGeekCleanUpService(IDataContext dataContext, IBoardGameGeekApiClient boardGameGeekApiClient, IRollbarClient rollbar)
+        {
+            _dataContext = dataContext;
+            _boardGameGeekApiClient = boardGameGeekApiClient;
+            _rollbar = rollbar;
+        }
+
+        public LinkOrphanGamesResult LinkOrphanGames()
+        {
+            var result = new LinkOrphanGamesResult();
+            var stopwatch = new Stopwatch();
```

Comment:
Sweet, I didn't know about this class :) Beats system.currentimeinmillis comparisons :+1: